pkg/curatedpackages/packagecontrollerclient.go

```diff
@@ -32,9 +41,11 @@ const (
 type PackageControllerClientOpt func(client *PackageControllerClient)
 
 type PackageControllerClient struct {
-	kubeConfig            string
-	chart                 *releasev1.Image
-	chartInstaller        ChartInstaller
+	kubeConfig     string
+	chart          *releasev1.Image
+	chartInstaller ChartInstaller
+	// deleter handles helm chart install deletion.
+	deleter               ChartInstallationDeleter
```

Comment:
Can we just call this `ChartUninstaller`?

Reply:
I guess? Helm has weird wording for a chart installation or whatever. I guess I don't have a strong feeling on it, it's just all confusing.